package.json

```diff
@@ -12,7 +13,7 @@
     "build": "turbo run build --concurrency=100%",
     "changelog:generate": "tsx scripts/changelog.ts",
     "clean": "turbo run clean --concurrency=100%",
-    "dev": "TSUP_SKIP_DTS=true turbo run dev --concurrency=100% --filter !@latticexyz/explorer",
+    "dev": "turbo run dev --concurrency=999",
```

Comment:
`dev` task uses `persistent: true` to mark it as a long running task (which helps warn when other tasks depend on a thing that will never complete)

thus, we need to manually specify concurrency > number of persistent tasks